File: Dart/CNDARTLANG/bin/Functions/PiEuler.dart
pisum() {
  var sum = 0.0;
  for (int j = 1 ; j <= 500 ; j++){
    sum = 0.0;
    for (int k = 1 ; k <= 10000 ; k++){
      sum = sum + 1.0/(k*k);
    }
  }
  return sum;
}

pisunvec() {
  var a = new List<int>.generate(10000, (i) => i + 1);
  double s = 0;

  for (int j = 0 ; j < 500 ; j++) {
    for (int k = 0; k < a.length; k++) {
      s += 1 / (a[k] ^ 2);
    }
  }
  return s;
}

File: Kotlin/CNKLANG/src/Functions/PiEuler.kt
package Functions

import kotlin.math.pow

fun pisum(): Double {
    var sum = 0.0
    val n = 500
    for(i in 0..n){
        sum = 0.0
        for(k in 1..(n*20))
        sum = sum + 1.0/(k*k)
    }
    return sum
}

fun pisunvec(): Double {
    val a = generateSequence(0.0, { 0.0 }).take(10000).toList()
    var s = 0.0
    val n = a.size/20

    for(i in 0..n){
        for(k in 0 until a.size)
        s += 1/(a[k].pow(2))
    }
    return s
}